### DISCUSSION TITLE
Formula days left

### DISCUSSION THREAD

**Discussion by DannyHatcher-Git** [RafaelGB/obsidian-db-folder#485] [Q&A]
I am using this: ${row.Deadline.diffNow("days").toFormat("d")}  days left but it isn't accurate as it seems to be 1 day off. I was wondering how I edit it to add a day.

**Comment by RafaelGB** [comment]
The object used for the plugins dates its the same of dataview, Luxon.

You can find information about this kind of object [here](https://moment.github.io/luxon/#/)

**Reply by DannyHatcher-Git** [reply]
So how does the formula match up to the dataview plugin? 
I guess a more specific question would be, how do I get a result in the formula property that I can show in a dataview query?

I love this plugin but explaining it to others is challenging.

**Reply by RafaelGB** [reply]
In the formulas section we would get into a more technical aspect.
The formulas do not match the dataview methodology and are still at a very early stage. At the moment it is limited to just displaying text using js expressions.

In the use case that we proposed, I commented that all Luxon functionalities would be available for date types.(it is quite complete) [The formating section](https://moment.github.io/luxon/#/formatting?id=formatting) of Luxon could be helpful in this case

**Reply by DannyHatcher-Git** [reply]
Awesome, thank you. I will have a look later!
